.github/actions/baseline-test/action.yml

```diff
@@ -0,0 +1,222 @@
+name: 'Hello World'
+description: 'Greet someone and record the time'
+inputs:
+  node-version:
+    description: 'node-version'
+    required: true
+    default: '16.x'
+  python-version: 
+    description: python-version
+    default: '3.7'
+    required: true
+  case-path: 
+    description: 'case-path'
+    required: true
+    default:  'success/case1'
+  node_env:
+    description: 'node_env'
+    required: false
+    default:  'testing'
+  jina_hubble_registry: 
+    description: 'jina_hubble_registry'
+    required: false
+    default:  'http://localhost:3000'
+  github-token: 
+    description: github-token
+    required: false
+outputs:
+  result:
+    description: 'result'
+    value: ${{ steps.finally.outputs.result }}
+runs:
+  using: composite
+  steps:
+      - name: Clone latest hubble code
+        uses: actions/checkout@v2
+        env:
+            NODE_ENV: ${{ inputs.node_env }}
+            JINA_HUBBLE_REGISTRY: ${{ inputs.jina_hubble_registry }}
+        with:
+          repository: jina-ai/hubble
+          token: ${{ secrets.JINA_DEV_BOT }}
+          ref: refs/heads/master
+          path: hubble
```

Comment:
We probably should rename the path to something that won't conflict with the original repo. like `__actions__hubble__`.